src/en/proposals/errant-vox-species.md

```diff
@@ -0,0 +1,94 @@
+
+[Discord thread where we are attempting to contain the vox talk](https://discord.com/channels/310555209753690112/1200498334587179018)
+
+
+## Design Pillars
+
+Disclaimer, all my info is secondhand. If you have any input, feel free to nitpick about the details.
+
+My best attempt to boil down (or am I re-interpreting?) what the vox are about, are these three pillars: Weird, Difficult, and Cooperative.
+Everything designed about/for them should aim to channel or at the very least not contradict these three ideas.
+
+
+#### Weird
+They are a weird species, which is perhaps a strange thing to specify in a scifi setting with moth people and space carps, but even so. They are probably cybernetic and their biology is particularly alien I mean who dies from oxygen?! They scream, dammit why do they scream?? Why are they drinking the welder fluid??? While care should be taken that their features are not (encouraging to be) annoying, it's fine if they are a little Weird.
+
+#### Difficult
+They are not intended to be perfectly balanced, or even necessarily what you might call "on par". Their main "feature" is in itself very punishing. If they overall fall towards the weaker side of the meta, that's just the cost of being vox. While they can and should have some positives, they should not be given big buffs just to "balance out" their shortcomings. They should not be prohibitively difficult, but if they are a less popular choice, that fits just fine into Difficult.
+
+#### Cooperative
+Historically, due to the difficulty of surviving a round as a vox, they had a tendency to rely on each other to get by in a world that was just not built for them. When no one else "got it", vox players turned for support to those who did. (Or so I'm told). It is the position of this design that this cooperative aspect of them is something to be preserved, and it being species specific is not something that should either be encouraged or discouraged mechanically, leaving it to the players to decide.
+Obviously, "just designating" how a species is supposed to play like is an impossible, lofty goal, but nevertheless their features could be designed in a way to help with that
+This was the hardest pillar to phrase and will be the hardest to come up with ideas for. But if a feature works better with other players, does something for other players rather than you, or is a flaw that is more easily counteracted with external help rather than just having the right tool in your pocket, then that feature is Cooperative.
+
+
+## Specific Features and Ideas 
+(All subject to further review during the actual implementation, of course. Especially towards the end of the list. They are mostly just intended as a showcase of ideas to start out with)
+
+  - There should be some sort of warning or confirmation when selecting them, so players, especially new players, don't end up unwittingly picking a choice that will kill their character for the wrong click. 
```

Comment:
Well, that was somewhat of an overstatement. But they do die in ~2 minutes (it was apparently half this in 13) if they take off their mask. 
Although this would be less of a noobtrap by the time I'm done implementing them than it would be right now, after they have both the automatic roundstart mask activation and toxin alerts. (Though you would still need to know what's the deal with the toxins, I can potentially see a clueless player who just picked vox take their mask off and then spend their remaining 2 minutes wondering what's going on and how they stop it)
Though in general I think it would be a QoL improvement if players got some info about each species at character creation, I have seen questions about that pop up on Discord often enough. The warning could simply be a part of that infocard

Reply:
Having species with varied gameplay and difficulty is a nice way to add more replayability and flavor. I don't think we should necessarily tone down difficulty for the sake of new players. 
I also agree that we definitely should have some sort of system of warning new players on the character/job selection screen about harder/unconventional roles/races. It might also be a good idea to have difficulties listed next to each species to make it clear to new players that there are gameplay variations between them